src/mlpack/tests/main_tests/linear_regression_test.cpp

```diff
@@ -41,19 +46,13 @@ struct LRTestFixture
   }
 };
 
-void ResetSettings()
-{
-  IO::ClearSettings();
-  IO::RestoreSettings(testName);
-}
-
-BOOST_FIXTURE_TEST_SUITE(LinearRegressionMainTest, LRTestFixture);
-
 /**
  * Training a model with different regularization parameter and ensuring that
  * predictions are different.
  */
-BOOST_AUTO_TEST_CASE(LRDifferentLambdas)
+
+TEST_CASE_METHOD(LRTestFixture, "LRDifferentLambdas",
+                 "LinearRegressionMainTest")
```

Comment:
```suggestion
TEST_CASE_METHOD(LRTestFixture, "LRDifferentLambdas",
                 "[LinearRegressionMainTest][BindingTests]")
```
@jeffin143 @zoq what do you think, add another tag like this?

Reply:
Yes, I like the idea.